externalAPI/tradeshiftAPI/supplierAPI.go

```diff
@@ -124,5 +127,58 @@ func (t *TradeshiftAPI) CreateOffer(name string, buyerID string) (string, error)
 	if err != nil {
 		return "", err
 	}
-	return r, err
+	return strings.Replace(r, "\"", "", -1), err
+}
+
+func (t *TradeshiftAPI) UpdateOffer(
+	offerID string,
+	name string,
+	startDate *time.Time,
+	endDate *time.Time,
+	countries []string) error {
+	method := fmt.Sprintf("/product-engine/supplier/supplier/v1/offers/%v", offerID)
+
+	status := "Draft"
+	message := "Hello\n\n... Go ahead and write a personal message to the receiver of this offer.\n\nBest regards"
```

Comment:
hardcoded this params here, in future they can be configurable in import file

Reply:
We don't need to have this hardcode here. They are optional fields.
If we have Name, Start Date, End date and Countries then JSON should look like
{"name":"test9","startDate":1626282000000,"endDate":1626282000000,"countries":["US","CA"]}

Reply:
fixed